### DISCUSSION TITLE
Select next argument in a function?

### DISCUSSION THREAD

**Discussion by Klaaktu** [zed-industries/zed#19451] [Feature Requests]
When typing a function, Zed automatically adds the signature inside the bracket and select the 1st argument. But is there a way to select the next argument with keyboard, instead of using mouse drag&select?

**Comment by AlessandroBellati** [comment]
Hi, you can press tab to move to the next argument and shift+tab to move to the previous.

**Reply by bbb651** [reply]
This only works when there are already tab stops, which only works right after the completion before they’re dismissed, and even then (at least for me, I assume it’s language server dependent) it only works for snippets and control flow not functions.

**Comment by bbb651** [comment]
There is a helix feature for this called smart tab which looks absolutely brilliant, please upvote the feature request: #14803

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
